dev-tools/reconfigurator-cli/src/lib.rs

```diff
@@ -837,6 +841,44 @@ struct LoadExampleArgs {
     /// Do not create entries for disks in the blueprint.
     #[clap(long)]
     no_disks_in_blueprint: bool,
+
+    /// Set a 0-indexed sled's policy
+    #[clap(long, value_name = "INDEX:POLICY")]
+    sled_policy: Vec<LoadExampleSledPolicy>,
+}
+
+#[derive(Clone, Debug)]
+struct LoadExampleSledPolicy {
+    /// The index of the sled to set the policy for.
+    index: usize,
+
+    /// The policy to set.
+    policy: SledPolicy,
+}
+
+impl FromStr for LoadExampleSledPolicy {
+    type Err = anyhow::Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        let Some((index, policy)) = s.split_once(':') else {
+            return Err(anyhow!("invalid format, expected <index>:<policy>"));
```

Comment:
Tiny nit - could shorten this to

```suggestion
            bail!("invalid format, expected <index>:<policy>");
```

or use `s.split_once(':').context("...")?;` to match the index parsing.